### DISCUSSION TITLE
Optimal torch placement overlay

### DISCUSSION THREAD

**Discussion by Delivator** [SchmarrnDevs/Lighty#84] [Ideas and Suggestions]
Add a marker to indicate the best spot to place a torch (or other light block) for most efficient placement to make an area completely spawnproof!

Also big props, love the mod so far!

**Comment by Delivator** [comment]
Oh sorry I just now see that there is an [issue for the same idea](https://github.com/SchmarrnDevs/Lighty/issues/54). I only looked at the ideas and suggestions here

**Comment by andi-makes** [comment]
No problem! And I'll have to confess, having everything split between Discussions and Issues and the Discord isn't really optimal and even confusing to me :upside_down_face:
